api/restHandler/WebhookListenerIFace.go

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020 Devtron Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package restHandler
+
+import (
+	"net/http"
+)
+
+type WebhookListenerIFace interface {
```

Comment:
As a convension we dont use IFace, we call it WebhookListener and its implementation will be WebhookListenerImpl as other classes deal with interface and not implementation

Reply:
done